braid/uvector.c

```diff
@@ -269,6 +312,21 @@ _braid_USetVector(braid_Core        core,
       _braid_BaseFree(core, app,  u);              /* free the vector */
    }
 
+   /* If braid is finished, make sure the last time point is stored, i.e.,
```

Comment:
@rfalgout @eric-c-cyr @steffi7574 
Hi All -- I just moved the storage logic for ulast into the USetVector routine below.  Per the discussion with Rob, I think this is a better place for it.  All the regression tests for Torchbraid past, so I think everything  is OK.  But feel free to double check, too.